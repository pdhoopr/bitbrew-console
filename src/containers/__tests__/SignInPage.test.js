import { shallow } from 'enzyme';
import React from 'react';
import ConnectedSignInPage from '../SignInPage';

const SignInPage = ConnectedSignInPage.wrappedComponent;

test('renders DOM for the sign in page', () => {
  const dom = shallow(<SignInPage signIn={jest.fn()} />);
  expect(dom).toMatchSnapshot();
});
