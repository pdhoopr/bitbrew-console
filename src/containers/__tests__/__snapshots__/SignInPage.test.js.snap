// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DOM for the sign in page 1`] = `
<UNDEFINED>
  <styled.header>
    <styled.div>
      <styled.svg />
    </styled.div>
  </styled.header>
  <styled.main>
    <SignInForm
      signIn={[MockFunction]}
    />
    <styled.p>
      Not a BitBrew customer yet?Â 
      <styled.a
        href="mailto:hello@bitbrew.com"
      >
        Contact us
      </styled.a>
      .
    </styled.p>
  </styled.main>
  <Footer />
</UNDEFINED>
`;
